src/Symfony/Component/Form/Extension/Core/DataTransformer/DateTimeToLocalizedStringTransformer.php

```diff
@@ -130,11 +130,11 @@ public function reverseTransform($value)
         try {
             if ($dateOnly) {
                 // we only care about year-month-date, which has been delivered as a timestamp pointing to UTC midnight
-                return new \DateTime(gmdate('Y-m-d', $timestamp), new \DateTimeZone($this->inputTimezone));
+                $dateTime = new \DateTime(gmdate('Y-m-d', $timestamp), new \DateTimeZone($this->outputTimezone));
```

Comment:
that's not my point ! for me it shoud be `$dateTime = new \DateTime(gmdate('Y-m-d', $timestamp))` so that the date i write/select in my form is the same as the one i get back in the code